sql/common/src/test/scala/org/apache/sedona/sql/dataFrameAPITestScala.scala

```diff
@@ -978,5 +978,14 @@ class dataFrameAPITestScala extends TestBaseScala {
       val actual = df.take(1)(0).getInt(0)
       assert(expected == actual)
     }
+
+    it("Passed ST_Translate") {
```

Comment:
Please add a test for the case that has no optional parameter.

Reply:
Done